docs/changelog.md

```diff
@@ -29,6 +29,19 @@ permalink: /changelog/
   * Added ability to show one item per search suggestion ([#1779](https://github.com/mozilla-mobile/android-components/issues/1779))
   * Added ability to define custom hooks to be invoked when editing starts or is completed.
 
+* **feature-browser**
+  * Added `BrowserToolbar` attributes to color the menu.
+  ```
+  <mozilla.components.browser.toolbar.BrowserToolbar
+      android:id="@+id/toolbar"
+      android:layout_width="match_parent"
+      android:layout_height="56dp"
+      android:background="#aaaaaa"
+      app:browserToolbarMenuColor="@color/photonBlue50"
+      app:browserToolbarInsecureColor="@color/photonRed50"
+      app:browserToolbarSecureColor="@color/photonGreen50" />
```

Comment:
I wonder if we need XML attributes or if we can define styles without them? (@Amejia481)

Reply:
We can define styles without XML attributes,  but they're handy for configuring single properties.

Reply:
Having both gives consumers more flexibility  of choosing which level of configuration they want

Reply:
That's definitely true. I was mostly concerned about having a gigantic XML API that we need to maintain.

Reply:
It's definitely not required like mentioned above. In this case, can't get the attribute set from `BrowserToolbar` in `DisplayToolbar` so most of the work is to enable access to the display toolbar, the rest is a nice-to-have. 🤷‍♂️